packaging/installer/netdata-updater.sh

```diff
@@ -352,6 +352,91 @@ disable_netdata_updater() {
   return 0
 }
 
+auto_update_status() {
+  case "$(_get_scheduler_type)" in
+    systemd) info "The default auto-update scheduling method for this system is: systemd timer units" ;;
+    crontab) info "The default auto-update scheduling method for this system is: drop-in crontab" ;;
+    interval) info "The default auto-update scheduling method for this system is: drop-in periodic script" ;;
+    *) info "No recognized auto-update scheduling method found" ; return ;;
+  esac
+
+  duplicate=""
+  enabled=""
+
+  if issystemd; then
```

Comment:
This fails if run as a regular user.

- regular user

```bash
pve-deb-work ~ $ ./netdata-updater.sh --auto-update-status 2>&1 | grep systemd
Fri Jan  3 02:59:09 PM EET 2025 : INFO: netdata-updater.sh:  Auto-updates using a systemd timer unit are NOT SUPPORTED due to: Systemd not present
```

- root

```bash
pve-deb-work ~ $ sudo ./netdata-updater.sh --auto-update-status 2>&1 | grep systemd
Fri Jan  3 02:59:12 PM EET 2025 : INFO: netdata-updater.sh:  Auto-updates using a systemd timer unit are NOT SUPPORTED due to: Required unit files not installed
```

Reply:
Should be resolved now (and the changes to fix this should in fact make all of our systemd detection both more reliable and faster).